benches/benches/vm_set/blockchain.rs

```diff
@@ -571,10 +598,19 @@ pub fn run(c: &mut Criterion) {
                 op::addi(0x15, 0x15, 1),
                 op::aloc(0x15),
                 op::move_(0x14, RegId::HP),
+                op::gtf_args(0x27, 0x00, GTFArgs::ScriptData),
+                op::addi(0x27, 0x27, ContractId::LEN.try_into().unwrap()),
+                op::addi(0x27, 0x27, WORD_SIZE.try_into().unwrap()),
+                op::addi(0x27, 0x27, WORD_SIZE.try_into().unwrap()),
+                op::addi(0x27, 0x27, AssetId::LEN.try_into().unwrap()),
```

Comment:
Can you provide some context on why we want these extra op codes on post call?

It looks like you're setting register `0x27`, which is the `$rc` in the upcoming `srwq` opcode, to the script data pointer, then bumping it by a contact ID, a word, a word, and lastly, an asset ID. Is this some sort of additional setup for the `srwq` at the next iteration?

Reply:
The `VmBench::contract_using_db` already sets some script data inside, so we need to skip data set by the `contract_using_db` to get access to the data that we set inside of the benchmark in line 614.

<img width="768" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/78b64254-b7b6-402b-a252-a30c29e98335">

<img width="908" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/e83815ef-8ba8-4252-a45d-12fd33d0678a">